doc/guide/Guide_Installation.ipynb

```diff
@@ -29,35 +29,39 @@
     "\n",
     "### Conda\n",
     "\n",
-    "It's highly recommanded to use [Conda](https://docs.conda.io/projects/conda/en/latest/) for the installation. Without Conda the installation of the dependencies can be cumbersome. If it is not already installed, download the latest version of [Anaconda](https://www.anaconda.com/) or [Miniconda](https://conda.io/miniconda.html) and execute it.\n",
+    "It's highly recommanded to install CLIMADA into a [Conda](https://docs.conda.io/projects/conda/en/latest/) environment. Without Conda, the installation of the dependencies can be cumbersome. If it is not already installed, download the latest version of [Anaconda](https://www.anaconda.com/) or [Miniconda](https://conda.io/miniconda.html) and execute it.\n",
     "\n",
     "### Install dependencies\n",
     "\n",
-    "Open a command prompt (Windows) or shell (Mac, Linux)\n",
+    "Open a command prompt (Windows) or shell (Mac, Linux) and run:\n",
     "\n",
     "```bash\n",
     "cd $[installation directory]\n",
     "conda env create -n climada_env -f climada_petals/requirements/env_climada.yml\n",
     "conda activate climada_env\n",
     "```\n",
     "\n",
-    "### Install climada core and petals\n",
+    "### Install CLIMADA's paths for core and petals\n",
+    "\n",
+    "Include the CLIMADA paths into the Conda environment as follows:\n",
```

Comment:
Since people often think that the word "install" means that some files are copied to some other location, I made the wording more clear. Otherwise, people might think that there are still paths to be set after installation.